cmd/root.go

```diff
@@ -201,10 +202,17 @@ func printCheckStart(enabledChecks checker.CheckNameToFnMap) {
 	}
 }
 
-func printProbeResults(enabledProbes []string) {
+func printProbeResults(enabledProbes []string, findings []finding.Finding) {
 	for _, probeName := range enabledProbes {
 		fmt.Fprintf(os.Stderr, "Finished probe %s\n", probeName)
 	}
+	for _, result := range findings {
+		if result.Remediation != nil {
```

Comment:
Can you add a comment here that we're doing this because it's an indicator of a bad probe outcome?

Reply:
```suggestion
		if result.Remediation != nil {
		    // Remediation is indicative of a bad result.
		    // This output is a light alternative to the details provided by --format=probe
```